export_weekly_advert_contacts.py

```diff
@@ -0,0 +1,89 @@
+import argparse
+import csv
+import json
+
+from core_data_modules.logging import Logger
+from id_infrastructure.firestore_uuid_table import FirestoreUuidTable
+from storage.google_cloud import google_cloud_utils
+
+from src.lib import PipelineConfiguration
+
+Logger.set_project_name("IMAQAL")
+log = Logger(__name__)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Generates lists of phone numbers of consented participants"
+                                                 "who participated in the last five episodes to be used in weekly "
+                                                 "sms advert")
+
+    parser.add_argument("google_cloud_credentials_file_path", metavar="google-cloud-credentials-file-path",
+                        help="Path to a Google Cloud service account credentials file to use to access the "
+                             "credentials bucket")
+    parser.add_argument("pipeline_configuration_file_path", metavar="pipeline-configuration-file",
+                        help="Path to Imaqal full pipeline configuration json file")
+    parser.add_argument("demog_map_json_input_dir", metavar="demog-map-json-input-dir",
+                        help="Path to a directory to read per episode demog map .json files. "
+                             "The files should be in the format <rqa_raw_field>_demog_map.json")
+    parser.add_argument("current_rqa_raw_field", metavar="current-rqa-raw-field",
```

Comment:
```suggestion
    parser.add_argument("advert_rqa_raw_field", metavar="current-rqa-raw-field",
```
I think this is clearer, as we tend to generate contacts to advertise to the week before the show itself actually airs